lldb/test/Shell/SymbolFile/DWARF/x86/module-ownership.mm

```diff
@@ -26,7 +26,7 @@
 // CHECK-ANON-S1: CXXRecordDecl {{.*}} imported in A struct
 
 StructB s3;
-// CHECK-ANON-S2: CXXRecordDecl {{.*}} imported in A.B {{.*}} struct
+// CHECK-ANON-S2: CXXRecordDecl {{.*}} imported in A.B {{.*}}struct
```

Comment:
In case someone asks about this: before we force-parsed the DIE by calling GetDeclContext() there used to be an `<undeserialized declaration>` output here that is irrelevant to the test.